include/ensmallen_bits/problems/levi_function_n13.hpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file levi_function_n13.hpp
+ * @author Suryoday Basak
+ *
+ * Definition of the Levi function N.13.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_PROBLEMS_LEVI_FUNCTION_N13_HPP
+#define ENSMALLEN_PROBLEMS_LEVI_FUNCTION_N13_HPP
+
+namespace ens {
+namespace test {
+
+/**
+ * The Levi function N.13, defined by
+ *
+ * \f[
+ * f(x_1, x_2) = sin(3 * pi * x_1)^2 + (x_1 - 1)^2 * (1 + (sin(3 * pi * x_2)^2)) +
+ * 	    	 (x_2 - 1)^2 * (1 + sin(2 * pi * x_2)^2)
+ * \f]
+ * This should optimize to f(x_1, x_2) = 0, at x = [1, 1].
+ *
+ * For more information, please refer to:
+ *
+ * @code
+ * @misc{LeviFunction,
+ *	 URL = {http://www.sfu.ca/~ssurjano/levy13.html},
+ * }
+ * @endcode
+ */
+class LeviFunctionN13
```

Comment:
Levi or Levy, I think there are more references for Levy, including the one you referenced, so maybe it makes sense to change the name?

Reply:
Changed to Levy! :)